public/css/styles.css

```diff
@@ -0,0 +1,673 @@
+  margin-left: 8.5em;
+  margin-bottom: auto;
+}
+.skills-para {
+  padding: 2em 0;
+}
+
+/*---portfolio section css---*/
+nav a {
+  text-decoration: none;
+  color: #686868;
+  font-weight: bold;
+  font-size: 1.15em;
+}
+nav ul {
+  margin-left: -2em;
+}
+
+nav a:hover {
+  color: #000000;
+}
+
+.portfolio-nav a:hover{
+  background-color: #FF6124;
+  color: #ffffff;
+}
+
+.portfolio-nav li{
+  margin-right: 1em;
+}
+
+.carousel-lrg {
+  width: 66.5em;
+}
+
+/*---Team and clients CSS---*/
+.team-title {
+  font-size: 1em;
+  font-weight: 300;
+  margin-top: -1.2em;
+}
+.team-li {
+  margin-left: -2em;
+}
+.button-contact {
+  margin-bottom: 4em;
+  background-color: #FF6124;
+  color: #fff;
+  border: none;
+  font-size: .8em;
+  padding: 1em 2.5em;
+  font-weight: 500;
+}
+.face-pic {
+  padding-bottom: 1em;
+}
+.team-para {
+  font-size: .9em;
+  margin: 1em;
+}
+.client-wrapper {
+  display: inline-flex;
+  display: -webkit-flex;
+  margin-left: 9%;
+  margin-right: 9%;
+  margin-bottom: 3em;
+}
+.client-section {
+  display: flex;
+  display: -webkit-flex;
+  text-align: left;
+  padding-left: 3em;
+  padding-right: 3em;
+  width: 100%;
+}
+.arrow {
+  padding-bottom: 2em;
+  margin-top: 4em;
+}
+.client-pic {
+  width: 29%;
+  height: 0%;
+  display: block;
+  min-width: 13%;
+  margin-top: 2em;
+}
+.client-name {
+  color: #FF6124;
+  padding-left: 1em;
+  margin-left: .7em;
+}
+.client-title {
+  font-weight: 400;
+  font-size: .8em;
+}
+.client-text {
+  margin-left: 2em;
+  padding-bottom: 2em;
+  margin-right: 0;
+}
+.client-text-wrapper {
+  margin-top: 1em;
+}
+
+/*---css for social media links/icon fonts ---*/
+@font-face {
+    font-family: 'socicon';
+    src: url('../font/socicon.eot');
+    src: url('../font/socicon.eot?#iefix') format('embedded-opentype'),
+         url('../font/socicon.woff') format('woff'),
+         url('../font/socicon.ttf') format('truetype'),
+         url('../font/socicon.svg#sociconregular') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}
+
+.socicon {
+    font-family: 'socicon';
+    color: #ffffff;
+    font-size: .9em;
+    padding: 0 .5em;
+}
+
+.social-media{
+  list-style-type: none;
+  margin-left: -2em;
+}
+
+.social-media a{
+  text-decoration: none;
+}
+
+.social-media li{
+  display: inline;
+}
+
+.social-media li:nth-of-type(1){
+  background-color: #3b5998;
+}
+.social-media li:nth-of-type(2){
+  background-color: #4099FF;
+}
+.social-media li:nth-of-type(3){
+  background-color: #ea4c89;
+}
+.social-media li:nth-of-type(4){
+  background-color: #007bb5;
+}
+.social-media li:nth-of-type(5){
+  background-color: #fffc00;
+}
+
+.social-media li:hover{
+  background-color: #FF6124;
+}
+
+/*---Contact us section css---*/
+.contact-us{
+  text-align: center;
+}
+
+.contact-us form{
+  color: #606060;
+  width: 100%;
+}
+
+.contact-us button{
+  color: #FF6124;
+  font-weight: 500;
+}
+
+.contact-us textarea{
+  color: #fff;
+  height: 10em;
+  width: 100%;
```

Comment:
This property is being set on `input, textarea` below on line 472. Don't repeat yourself. Prefer the unqualified selector.